staging/src/k8s.io/api/admissionregistration/v1/types.go

```diff
@@ -114,7 +114,7 @@ const (
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// ValidatingWebhookConfiguration describes the configuration of and admission webhook that accept or reject and object without changing it.
+// ValidatingWebhookConfiguration describes the configuration of admission webhook that accept or reject object without changing it.
```

Comment:
I think both "and" should be "an". See https://github.com/kubernetes/kubernetes/commit/9ddea83a2ce0937cf0fc8f8c35614bb18e74cfad#diff-0c58bb8027e5eeb97b7d77d32c58a9def5816e61d141c4daf92bf351c707e91cR121

Reply:
Maybe "a list of" is more accurate as the field **Webhooks** is an array?

Reply:
"a list of admission webhooks" sounds fine to me